docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -607,3 +613,27 @@ Remote Locators Allocations
        traffic. See :ref:`remotelocatorsallocationattributes`.
      - ``uint32_t``
      - 1
+
+.. _SendBuffers:
+
+Send buffers
+************
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Name
+     - Description
+     - Values
+     - Default
+   * - ``<preallocated_number>``
+     - Initial number of send buffers to allocate. See |br|
+       :ref:`sendbuffersallocationattributes`.
+     - ``uint32_t``
+     - 0
```

Comment:
I think they are limited.

preallocated_number: 

This attribute controls the initial number of send buffers to be allocated. The default value of 0 will perform an initial guess of the number of buffers required, based on the number of threads from which a send operation could be started.

dynamic:

When true, a new buffer will be created. When false, it will wait for a buffer to be returned. This is a trade-off between latency and dynamic allocations.